src/stories/Popover.stories.tsx

```
@@ -1,9 +1,12 @@
 import Popover from '@/components/Popover';
 import { useArgs } from '@storybook/preview-api';
 import { Meta, StoryObj } from '@storybook/react';
+import { ComponentProps } from 'react';
```

ESLint

[error] 'ComponentProps' is defined but never used.

(@typescript-eslint/no-unused-vars)